products/visionos.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Vision OS
+category: os
+tags: apple
+iconSlug: apple
+permalink: /visionos
+releasePolicyLink: https://en.wikipedia.org/wiki/VisionOS#Version_history
+eoasColumn: true
+releaseDateColumn: true
+
+releases:
+-   releaseCycle: "1"
+    releaseDate: 2024-02-02
+    eoas: false
+    eol: false
+    latest: "1.2"
+    latestReleaseDate: 2024-06-10
+    link: https://support.apple.com/en-us/118202
+
+---
+
+> [visionOS](https://www.apple.com/visionos), Apple's first spatial operating system, places digital content in a user’s physical world while allowing them to stay present and connected to others. It has an infinite canvas > for apps that scales beyond the boundaries of a traditional display and introduces a fully three-dimensional user interface controlled by the most natural and intuitive inputs possible — a user’s eyes, hands, and voice. > visionOS brings a new dimension to powerful, personal computing by changing the way users interact with their favorite apps, capture and relive memories, enjoy stunning TV shows and movies, and connect with others.
```

Comment:
The description is a bit too much "commercial". Something more neutral (such as https://en.wikipedia.org/wiki/VisionOS#Version_history) would be better.

Reply:
I have updated the description based on the wiki, let me know if it looks better.